jujugui/static/gui/src/app/components/env-switcher/env-switcher.js

```
@@ -194,7 +194,7 @@ YUI.add('env-switcher', function() {
       // We need to coerce error types returned by JES vs JEM into one error.
       var err = data.err || error;
       if (err) {
-        console.log(err);
+        console.log('error creating new model:', err);
```

Indeed, we should really return the error to the caller, I'll add an XXX.